src/app/static/src/app/components/ResetConfirmation.vue

```diff
@@ -59,32 +66,48 @@
 
     interface Data {
         waitingForVersion: boolean
+        versionNote: string
     }
 
     export default Vue.extend<Data, unknown, Computed, any>({
         props: ["open", "continueEditing", "cancelEditing"],
         data: function () {
             return {
-                waitingForVersion: false
+                waitingForVersion: false,
+                versionNote: ""
             }
         },
         computed: {
             changesToRelevantSteps: mapGetterByName("stepper", "changesToRelevantSteps"),
             currentVersionId: mapStateProp<ProjectsState, string | null>("projects", state => {
                 return state.currentVersion && state.currentVersion.id;
             }),
+            currentVersionNote: mapStateProp<ProjectsState, string | null>("projects", state => {
+                return state.currentVersion?.note || "";
+            }),
             ...mapGetters(["isGuest"]),
             errorsCount: mapStateProp<ErrorsState, number>("errors", state => {
                 return state.errors ? state.errors.length : 0;
-            })
+            }),
+            handleVersionNote: {
+                get() {
+                    return this.currentVersionNote!
+                },
+                set(note: string) {
+                    this.versionNote = note;
+                }
+            }
         },
         methods: {
             handleConfirm: function () {
                 if (this.isGuest) {
                     this.continueEditing();
                 } else {
                     this.waitingForVersion = true;
-                    this.newVersion();
+                    if(this.currentVersionNote && !this.versionNote) {
+                        this.versionNote = this.currentVersionNote
+                    }
+                    this.newVersion(this.versionNote)
```

Comment:
Let's be consistent with semicolons

Reply:
This is TypeScript, semicolons are  highly unnecessary .

Reply:
Unnecessary, but we use them as standard, and we should really be consistent one way or the other.